ADRs/cloud-providers.md

```diff
@@ -0,0 +1,31 @@
+# Title
+
+Cloud Provider
+
+## Status
+
+Proposed
+
+## Context
+
+There are a myriad of cloud providers offering a wide range of services. It’s necessary to evaluate each provider.
```

Comment:
It’s necessary to evaluate each provider and choose the one which best suites the Spotlight platforms use cases